app/client/src/components/isochrones/Isochrones.vue

```diff
@@ -1528,6 +1534,50 @@ export default {
                 surfaceData: singleValuedSurface,
                 feature: olFeatures[0]
               };
+              let calcTimeFrom = calculation.config.settings.from_time;
+              let calcTimeTo = calculation.config.settings.to_time;
+
+              let calculatedTimeFrom = "";
+              if (calcTimeFrom % 3600 !== 0) {
+                let temp = calcTimeFrom / 3600;
+                let hours = parseInt(temp);
+                let minutes = Math.round((temp - hours) * 60);
+                if (hours / 10 < 1) {
+                  calculatedTimeFrom = `0${hours}:${minutes}`;
+                } else {
+                  calculatedTimeFrom = `${hours}:${minutes}`;
+                }
+              } else {
+                let temp = calcTimeFrom / 3600;
+                if (temp / 10 < 1) {
+                  calculatedTimeFrom = `0${temp}:00`;
+                } else {
+                  calculatedTimeFrom = `${temp}:00`;
+                }
+              }
+
+              let calculatedTimeTo = "";
+              if (calcTimeTo % 3600 !== 0) {
+                let temp = calcTimeTo / 3600;
+                let hours = parseInt(temp);
+                let minutes = Math.round((temp - hours) * 60);
+                if (hours / 10 < 1) {
+                  calculatedTimeTo = `0${hours}:${minutes}`;
+                } else {
+                  calculatedTimeTo = `${hours}:${minutes}`;
+                }
+              } else {
+                let temp = calcTimeTo / 3600;
+                if (temp / 10 < 1) {
+                  calculatedTimeTo = `0${temp}:00`;
+                } else {
+                  calculatedTimeTo = `${temp}:00`;
+                }
```

Comment:
I think here we can use the js Date() object to convert from seconds to hh:mm. Also it would better to create a helper function for this which we can put in Helpers.js, so we avoid duplications.

```
 const fromTimeDate = new Date(0);
      fromTimeDate.setSeconds(this.routingProfiles.transit.from_time);
      this.publicTransport.fromTime = fromTimeDate
        .toISOString()
        .substring(11, 16);
```